csds-client/client/client_test.go

```diff
@@ -132,7 +133,7 @@ func TestParseResponseWithoutNodeId(t *testing.T) {
 		t.Errorf("Read From File Failure: %v", err)
 	}
 	out := captureOutput(func() {
-		if err := printOutResponse(&response, ""); err != nil {
+		if err := printOutResponse(&response, "", false, false); err != nil {
 			t.Errorf("Print out response error: %v", err)
```

Comment:
This false, false seems not very clear. Let's have a struct to hold the options.

Reply:
Modified this to `c.info`